test/libsolidity/syntaxTests/nameAndTypeResolution/405_address_checksum_type_deduction.sol

```diff
@@ -1,9 +0,0 @@
-contract C {
-    function f() public {
-        var x = 0xfA0bFc97E48458494Ccd857e1A85DC91F7F0046E;
```

Comment:
Do we have a test that does e.g. `(0xfA0bFc97E48458494Ccd857e1A85DC91F7F0046E).send(2)`?

Reply:
Or better even: transfer

Reply:
`test/libsolidity/syntaxTests/nameAndTypeResolution/413_address_methods.sol` is the closest match I found, there, I could add a line at the bottom `(0xfA0bFc97E48458494Ccd857e1A85DC91F7F0046E).transfer(1);` - but that wouldn't necessarily correlate to this PR, IMHO (split?)

Reply:
Please change this test to use `(0xfA0bFc97E48458494Ccd857e1A85DC91F7F0046E).transfer(1)` because that is what it was supposed to test.